docs/source/command_line.rst

```diff
@@ -371,21 +371,24 @@ Here are some more useful flags:
   written by quick mode.
 
 - ``--python-executable EXECUTABLE`` will have mypy collect type information
-  from PEP 561 compliant packages installed for the Python executable
-  ``EXECUTABLE``. If not provided, mypy will use PEP 561 compliant packages
-  installed for the Python executable running mypy. See
-  :ref:`installed_packages` for more on making PEP 561 compliant packages.
+  from `PEP 561 <https://www.python.org/dev/peps/pep-0561/>`_ compliant
+  packages installed for the Python executable ``EXECUTABLE``. If not provided,
+  mypy will use PEP 561 compliant packages installed for the Python executable
+  running mypy. See :ref:`installed_packages` for more on making PEP 561
+  compliant packages. This flag will attempt to set ``--python-version`` if not
+  already set.
 
 - ``--python-version X.Y`` will make mypy typecheck your code as if it were
   run under Python version X.Y. Without this option, mypy will default to using
   whatever version of Python is running mypy. Note that the ``-2`` and
   ``--py2`` flags are aliases for ``--python-version 2.7``. See
   :ref:`version_and_platform_checks` for more about this feature. This flag
   will attempt to find a Python executable of the corresponding version to
-  search for PEP 561 compliant packages. If you'd like to disable this, see
-  ``--no-site-packages`` below.
+  search for `PEP 561 <https://www.python.org/dev/peps/pep-0561/>`_ compliant
+  packages. If you'd like to disable this, see ``--no-site-packages`` below.
 
-- ``--no-site-packages`` will disable searching for PEP 561 compliant packages.
+- ``--no-site-packages`` will disable searching for 
+  `PEP 561 <https://www.python.org/dev/peps/pep-0561/>`_ compliant packages.
```

Comment:
rst lets you define the link destination once at the bottom, and link to it multiple times with just `` `PEP 561`_ ``